cloudbuild.yaml

```diff
@@ -0,0 +1,68 @@
+timeout: 3600s # 1 hour
+logsBucket: 'gs://oppia-cloudbuild-logs'
+steps:
+  - id: build-oppia-base
+    name: gcr.io/kaniko-project/executor:latest
+    wait_for: ["-"]
+    args: 
+    - --dockerfile=Dockerfile.base
+    - --destination=gcr.io/oppia-jenkins-test-server/oppia-base:$SHORT_SHA
+    - --cache=true
+    - --cache-ttl=24h
+
+  - id: install-dependencies
+    wait_for: ["build-oppia-base"]
+    name: gcr.io/oppia-jenkins-test-server/oppia-base:$SHORT_SHA
```

Comment:
Where are we using Jenkins? I don't see it in the diff

Reply:
The project we were using to test this out was called oppia-jenkins-test-server even though we're not using Jenkins. Now that you mention it, we should probably parametrize the project name instead of hard coding.

Reply:
+1, I need to create a new server for this (or maybe rename the existing one).